src/components/Dropdown/index.js

```diff
@@ -0,0 +1,122 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import PropTypes from 'prop-types';
+import { Manager, Reference, Popper } from 'react-popper';
+
+import Overlay from '../Overlay';
+import { StyledDropdown } from './styles';
+
+const Dropdown = ({
+  children,
+  clickOutsideToHide,
+  onHide,
+  onOpen,
+  portalNode,
```

Comment:
why not include `modifiers` here?

Reply:
just to save 1 like of code 😅. I like the explicitness of `props.something` and it saves space xD